apparmor.d/groups/lxqt/lxqt-about

```diff
@@ -0,0 +1,31 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2024 Alexandre Pujol <alexandre@pujol.io>
+# Copyright (C) 2024 Besanon  <m231009ts@mailfence.com>
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/3.0>,
+
+#include <tunables/global>
+
+@{exec_path} = @{bin}/lxqt-about
+profile lxqt-about @{exec_path} {
+  include <abstractions/base>
+  include <abstractions/gtk>
+  include <abstractions/video>
+  include <abstractions/lxqt>
+      
+  @{exec_path}			mr,
+
+  /usr/share/icons/* 			r,
+  /usr/share/icons/{,**}  		r,
```

Comment:
`/usr/share/icons/{,**}` is already included in `abstractions/freedesktop.org` (wich is in `abstractions/lxq`).